src/openklant/conf/api.py

```diff
@@ -4,10 +4,8 @@
 # Remove the reference - we don't have a single API version.
 del API_VERSION  # noqa
 
-KLANTEN_API_VERSION = "1.0.0"
-CONTACTMOMENTEN_API_VERSION = "1.0.0"
-KLANTINTERACTIES_API_VERSION = "0.0.4"
-CONTACTGEGEVENS_API_VERSION = "1.0.0"
+KLANTINTERACTIES_API_VERSION = "0.1.0"
+CONTACTGEGEVENS_API_VERSION = "1.1.0"
```

Comment:
I think the schema needs to be generated as well. If I generate it locally I see more a diff, so I think the job to check if the schema was generated isn't running properly